src/50_Visual_Effects/Snackbar_%2F_Toast.html

```diff
@@ -0,0 +1,107 @@
+<!---{
+    "preview": "default"
+}--->
+<!--
+  ## Introduction
+
+  [Snackbars](https://material.io/guidelines/components/snackbars-toasts.html) provide brief feedback about an operation through a message at the bottom of the screen.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="<%host%>/../visual_effects/snackbar/toast/">
```

Comment:
is the name snackbar/toast/? is there a / between snackbar and toast?

Reply:
https://material.io/guidelines/components/snackbars-toasts.html

it's the same thing

Reply:
Renamed the sample